src/utils/urls.js

```diff
@@ -49,15 +49,16 @@ export const externalNavigation = (url) => {
   // eslint-disable-next-line no-undef
   window.location = url
 }
-export const baseURL = 'https://tab.gladly.io/cats/'
-export const getReferralUrl = (username) => {
+export const constructBaseUrl = (landingPagePath) =>
+  `https://tab.gladly.io${landingPagePath}`
+export const getReferralUrl = (username, landingPagePath) => {
   const referralUrl = username
-    ? `${baseURL}?u=${encodeURIComponent(username)}`
-    : baseURL
+    ? `${constructBaseUrl(landingPagePath)}?u=${encodeURIComponent(username)}`
+    : constructBaseUrl(landingPagePath)
```

Comment:
Could you add some tests for this? Mainly, as defensive programming: if landingPagePath is undefined, make sure it correctly links to the main homepage.

Reply:
ok.  I was building everything to default to cats but main homepage makes sense